hsr_navigation/Points[] data